build.gradle

```diff
@@ -508,12 +523,25 @@ task unzipPerfTop(type: Copy) {
     enabled = !file(perfTopBin).exists()
 }
 
-task enablePaAndRca(type: Exec) {
+task enablePa(type: Exec) {
     dependsOn(waitForES)
+    doFirst {
+        sleep(5000)
+    }
     commandLine('curl', 'localhost:9200/_opendistro/_performanceanalyzer/cluster/config', '-H',
             'Content-Type: application/json', '-d', '{"enabled": true}')
+    doLast {
+        sleep(5000)
```

Comment:
Should We add a comment explaining the rationale for the sleep ?

Reply:
Added